sphinx/application/network/concepts/queries/concept-network-query-save-profile/src/main/java/chat/sphinx/concept_network_query_save_profile/model/PersonInfoDto.kt

```diff
@@ -0,0 +1,15 @@
+package chat.sphinx.concept_network_query_save_profile.model
+
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class PersonInfoDto(
```

Comment:
Rename to `SavePeopleProfileDto` or similar since `PersonInfoDto` already exists on another module